conda/base/context.py

```diff
@@ -634,13 +634,6 @@ def root_prefix(self):
     def conda_prefix(self):
         return abspath(sys.prefix)
 
-    @property
-    # This is deprecated, please use conda_exe_vars_dict instead.
-    def conda_exe(self):
-        bin_dir = 'Scripts' if on_win else 'bin'
-        exe = 'conda.exe' if on_win else 'conda'
-        return join(self.conda_prefix, bin_dir, exe)
-
```

Comment:
So while this may have been marked for a long time as deprecated, I think we should still take it through the deprecation cycle as if we are just now starting the process